experimental/traffic-portal/src/app/core/profiles/profile-table/profile-table.component.ts

```diff
@@ -17,22 +17,28 @@ import { FormControl, UntypedFormControl } from "@angular/forms";
 import { MatDialog } from "@angular/material/dialog";
 import { ActivatedRoute, Params } from "@angular/router";
 import { BehaviorSubject } from "rxjs";
-import { ResponseProfile } from "trafficops-types";
+import { ProfileImport, ResponseProfile } from "trafficops-types";
 
 import { ProfileService } from "src/app/api";
+import { ExportAttachmentService } from "src/app/api/export-attachment.service";
 import { CurrentUserService } from "src/app/shared/current-user/current-user.service";
 import { DecisionDialogComponent } from "src/app/shared/dialogs/decision-dialog/decision-dialog.component";
-import { ContextMenuActionEvent, ContextMenuItem } from "src/app/shared/generic-table/generic-table.component";
+import { FileUtilsService } from "src/app/shared/file-utils.service";
+import { ContextMenuActionEvent, ContextMenuItem, TableTitleButton } from "src/app/shared/generic-table/generic-table.component";
+import { ImportJsonEditTxtComponent } from "src/app/shared/import-json-edit-txt/import-json-edit-txt.component";
 import { NavigationService } from "src/app/shared/navigation/navigation.service";
 
 /**
  * ProfileTableComponent is the controller for the profiles page - which
  * principally contains a table.
  */
 @Component({
+	providers: [
+		FileUtilsService
+	],
```

Comment:
You shouldn't need this, it's in the shared module, which is imported by the core module to which this component belongs